src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -135,6 +137,58 @@ function mergeParamsWithLocalACHData(data) {
     return updatedACHData;
 }
 
+function getVBBADataForOnyx() {
+    return {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
```

Comment:
Forgive me for asking "dumb" onyx questions, but I don't work in this code very much.

In what cases would `onyxMethod` and `key` be different for `optimisticData`, `successData` and `failureData`? It seems like these would always be the same? Maybe a failure would clear out an existing key?

Reply:
At least in most cases, they'll be the same, but sometimes a failure could also modify a `_DRAFT` key. I don't think I've seen any cases where the methods differ, but maybe a failure could clear out a key in certain cases (although, in most cases, it'll just clean up the content of the key)